### ISSUE TITLE
Send Normal_11bits frame receive Normal_29bits frame

### ISSUE THREAD

**Issue by DanielKupiniak** [pylessard/python-can-isotp#28]
Hi, is it possible to mix can id types for one stack? I want to send a standard frame but receive extended.

**Comment by pylessard** [comment]
Hi, it is not possible to do so right now. 

A quick workaround would be to convert the 29 bits ID to an arbitrary 11bits ID that the stack will accept (or the other way around).

I will look a the feasability to implement this during the weekend.

**Comment by DanielKupiniak** [comment]
Thanks for the tip, I will try to use it

**Comment by DanielKupiniak** [comment]
What do you think to create a class:
```
class Address.CanID(object):
     self.id
     self.addressingMode or self.is_extended
```
I know it will break interface, but also will limit the number of arguments to Address cls
What I see from the code and thanks to its modularity changes needs to be applied only in Address cls
<Good Job>

**Comment by pylessard** [comment]
Hi, there seems to be no gracveful solution to this that won't break the interfaces.
I will come up with something, but it may take some time. Sorry if there's a delay, I don't have much time now.

**Comment by pylessard** [comment]
I'm looking into this now. Will be possible in v2.0

**Comment by pylessard** [comment]
This is implemented in PR #107

**Comment by pylessard** [comment]
Fixed by #107. Will be part of v2.x
Release will be done in January 2024
